recipes/graspologic-native/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "graspologic-native" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graspologic_native-{{ version }}.tar.gz
+  sha256: 72b7586028a91e9fef9af0ef314d368f0240c18dca99e6e6c546334359a8610a
+
+build:
+  script:
+    - cargo bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - cargo-bundle-licenses
+    - python                     # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - maturin >=1.4,<2.0                     # [build_platform != target_platform]
+    - {{ compiler("c") }}
+    - {{ compiler("rust") }}
+  host:
+    - pip
+    - python
+    - maturin >=1.4,<2.0
+  run:
+    - python
```

Comment:
You also need to add a run dependency on __osx for the deployment target here, see https://conda-forge.org/news/2024/01/29/rust-175-osx-sdk/